### DISCUSSION TITLE
Wie kann ich die Verisonsnummer eines AddOns ermitteln und mit einer bestimmten Nummer vergleichen?

### DISCUSSION THREAD

**Discussion by tyrant88** [FriendsOfREDAXO/.github#41] [Q&A] [answered]
Wenn man im Code prüfen möchte, ob ein AddOn schon in einer bestimmten Version installiert ist, wie geht das?

**Comment by tyrant88** [comment]
Du kannst das via rex_string machen.

rex_string::versionCompare($addon->getVersion(), '1.2.3', '>')

thx @eaCe

**Reply by gharlan** [reply]
Seit 5.10 ist `rex_string::versionCompare` deprecated und es gibt neu `rex_version::compare`.
Wenn man also nicht kompatibel zu alten REDAXO-Versionen sein muss/möchte, sollte man eher die neue nutzen.

**Comment by tyrant88** [comment, marked as answer]
Also:
rex_version::compare($addon->getVersion(), '1.2.3', '>')

**Reply by eaCe** [reply]
Uff, mein Fehler
